ui/src/app/pages/homepage/recent-work.spec.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 
```

Comment:
Update this filename as well?

Reply:
noticed most of these immediately after pressing the re-request review button. On it, thank you